### ISSUE TITLE
Theme not installing correctly

### ISSUE THREAD

**Issue by tuitruth** [rulnoveid/tbloggerstrap#4]
When I installed this theme, it looks like this:
<img width="959" alt="Capture" src="https://github.com/rulnoveid/tbloggerstrap/assets/146628002/04fc5a8a-1690-4787-9d18-c2518fe41e0d">

**Comment by DealsBeam** [comment]
Also, the page speed need to be looked at https://pagespeed.web.dev/analysis/https-bstrap-inputekno-com/k9bcmbw23q?form_factor=mobile

**Comment by tuitruth** [comment]
> When I installed this theme, it looks like this: <img alt="Capture" width="959" src="https://private-user-images.githubusercontent.com/146628002/289017108-04fc5a8a-1690-4787-9d18-c2518fe41e0d.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM0NTgwNzgsIm5iZiI6MTcwMzQ1Nzc3OCwicGF0aCI6Ii8xNDY2MjgwMDIvMjg5MDE3MTA4LTA0ZmM1YThhLTE2OTAtNDc4Ny05ZDE4LWMyNTE4ZmU0MWUwZC5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyNFQyMjQyNThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZjU0YTI5Nzg0NDgzNTJkOWYzMDQwM2VjZTcxMzlmNzQyMTg5YjQwZTcwYmQyNmYyMjJkY2U0YzcxY2U2ZTgyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Eli7a9sU25qXBOJK3il2E9kteO7Dv7rzEEQVlPMYjrs">

I've managed to fix it. Saw the previous ticket.

**Comment by tuitruth** [comment]
> Also, the page speed need to be looked at https://pagespeed.web.dev/analysis/https-bstrap-inputekno-com/k9bcmbw23q?form_factor=mobile

Yes, isn't fantastic. I think part of the issue is Blogger/BlogSpot in general.

**Comment by rulnoveid** [comment]
Maybe I'll release an update to this template after the new year. I'm not sure about releasing it now. See the latest demo here https://bstraptemplate.blogspot.com/

**Comment by rulnoveid** [comment]
> > Also, the page speed need to be looked at https://pagespeed.web.dev/analysis/https-bstrap-inputekno-com/k9bcmbw23q?form_factor=mobile
> 
> Yes, isn't fantastic. I think part of the issue is Blogger/BlogSpot in general.

Maybe I'll release an update to this template after the new year. I'm not sure about releasing it now. See the latest demo here https://bstraptemplate.blogspot.com/

**Comment by DealsBeam** [comment]
> > > Also, the page speed need to be looked at [pagespeed.web.dev/analysis/https-bstrap-inputekno-com/k9bcmbw23q?form_factor=mobile](https://pagespeed.web.dev/analysis/https-bstrap-inputekno-com/k9bcmbw23q?form_factor=mobile)
> > 
> > 
> > Yes, isn't fantastic. I think part of the issue is Blogger/BlogSpot in general.
> 
> Maybe I'll release an update to this template after the new year. I'm not sure about releasing it now. See the latest demo here [bstraptemplate.blogspot.com](https://bstraptemplate.blogspot.com/)

Maybe these can help?

https://github.com/zkreations/hamlet

https://github.com/Rawandev-top/UltraSpeed

https://github.com/nagahitoyuki/imaginary-grid
